idseq_dag/steps/blast_contigs.py

```diff
@@ -205,16 +306,57 @@ def update_read_dict(read2contig, blast_top_m8, read_dict, accession_dict):
                 read2blastm8[read_id] = m8_line
         return (consolidated_dict, read2blastm8, contig2lineage, added_reads)
 
+
     @staticmethod
-    def run_blast(assembled_contig, reference_fasta, db_type, blast_m8, blast_top_m8):
+    def run_blast_nt(db_type, assembled_contig, reference_fasta, blast_m8, blast_top_m8):
```

Comment:
Is it standard practice to pass ie `db_type` as a parameter to this function even though by default it should only be using one DB type now that the `run_blast_nt` and `run_blast_nr` functions are separated? 
Same q. about the function `run_blast_nr` below.

Reply:
Great observation.   This `db_type` param is used only to compute the `blast_index` path.  That's duplicated code between the `run_blast_nt` and `run_blast_nr` functions, which should be pulled out into their caller.   Will update this PR shortly.

Reply:
Updated PR with fix based on this observation.